test/screens/alerts/alert_test.exs

```diff
@@ -16,7 +16,27 @@ defmodule Screens.Alerts.AlertTest do
     "header" => "Route 1 experiencing delays up to 20 minutes due to an accident.",
     "image" => nil,
     "image_alternative_text" => nil,
-    "informed_entity" => [%{"activities" => ~w[BOARD EXIT RIDE], "route" => "1"}],
+    "informed_entity" => [
```

Comment:
This is intended to be the _minimal_ valid attributes as per the naming and comment above. Rather than adding data for testing specific functionality to the "base" attributes that apply to all test cases, we should add the relevant attributes only within the test case, as in e.g. the `parses related facilities` test. (This could be a new `combines informed entities by direction_id` test.)